app/javascript/components/HtmlEditor.js

```diff
@@ -1,3 +1,22 @@
+{/*<HtmlEditor
```

Comment:
* It looks like this file isn't used anymore - it's been merged into the PostEditor as planned. Can delete.
* This line specifically: Is this documentation? Don't wrap in `{}` if you don't want to create a random JS block.